sql/bcnm_battlefield.sql

```diff
@@ -801,6 +801,31 @@ INSERT INTO `bcnm_battlefield` VALUES (512,3,17621010,0); -- Ancient Warrior
 INSERT INTO `bcnm_battlefield` VALUES (512,3,17621011,0); -- Ancient Warrior
 INSERT INTO `bcnm_battlefield` VALUES (512,3,17621012,0); -- Ancient Warrior
 INSERT INTO `bcnm_battlefield` VALUES (512,3,17621013,0); -- Ancient Warrior
+INSERT INTO `bcnm_battlefield` VALUES (515,1,17621248,3); -- Infernal Swarm
```

Comment:
All of these can be annotated with what they are, see the BCNM above

Reply:
Didn't annotate it like that as it was the only other BCNM I could find that was annotated that way. The rest declare the BCNM / KSNM once and list the values below. Is this the standard you want me to follow?

Reply:
We want things to be as readable as possible, so if the name isn't built into the SQL entry itself, it should probably be added as a comment etc.

Reply:
alright you bet!